--- git commit ---
2af73c4e46eaf0ce7a8423b5e3b691cdae064c4c


--- git status ---
On branch jump_test
Your branch is up to date with 'origin/jump_test'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   experiments/genesis/wheel_leg_infantry/tasks/jump/config_25cm.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/experiments/genesis/wheel_leg_infantry/tasks/jump/config_25cm.py b/experiments/genesis/wheel_leg_infantry/tasks/jump/config_25cm.py
index 2b80a83..f2d5ff0 100644
--- a/experiments/genesis/wheel_leg_infantry/tasks/jump/config_25cm.py
+++ b/experiments/genesis/wheel_leg_infantry/tasks/jump/config_25cm.py
@@ -246,6 +246,7 @@ def get_cfgs(locomotion_cfgs: Sequence[Mapping] | None = None):
                         "tracking_ang_vel": 20.0,
                         "flight_balance": 20.0,
                         "flight_airtime": 15.0,
+                        "flight_leg_vertical": 20.0,
                     },
                 },
             },
@@ -264,6 +265,7 @@ def get_cfgs(locomotion_cfgs: Sequence[Mapping] | None = None):
                         "tracking_ang_vel": 30.0,
                         # 腾空
                         "flight_balance": 30.0,
+                        "flight_leg_vertical": 30.0,
                         # 落地
                         "soft_landing": 50.0,
                         "landing_stability": 50.0,